misc.php

```diff
@@ -178,3 +178,10 @@ function wpcom_vip_wp_link_query_args( $query ) {
 }
 
 add_filter( 'wp_link_query_args', 'wpcom_vip_wp_link_query_args', 10, 1 );
+
+// Prevent Jetpack version ping-pong when a sandbox has an old version of stacks
+if ( true === WPCOM_SANDBOXED ) {
+	add_action( 'updating_jetpack_version', function() {
```

Comment:
`PHP_MAX_INT * -1` 😉 

Okay, that might be _too_ early, but earlier is definitely better here.